### PR TITLE
Migrated a bunch of unit tests to handleForTest

### PR DESCRIPTION
**Pull request by csilvestrini**
Just picked a bunch of easy ones. There's still lots more to go...

### CODE REVIEW

File: src/runtime/tests/arc-test.ts [hunk 1]
```diff
@@ -718,17 +727,18 @@ describe('Arc ' + storageKeyPrefix, () => {
...
 
     // TODO: Reference Mode: Deal With It (TM)
     varStore.referenceMode = false;
     colStore.referenceMode = false;
 
     // Populate the stores, run the arc and get its serialization.
     // TODO: the serialization roundtrip re-generates keys using the entity ids; we should keep the actual keys
-    await getSingletonHandle(varStore).set(new dataClass({value: 'v1'}));
+    await varHandle.set(new dataClass({value: 'v1'}));
     await colStore.store({id: 'i2', rawData: {value: 'v2', size: 20}}, ['i2']);
```
Comment (galganif) [inline src/runtime/tests/arc-test.ts]: can colStore be migrated to colHandle as well?

File: src/runtime/tests/description-test.ts [hunk 2]
```diff
@@ -473,7 +473,7 @@ recipe
     `));
 
       // Add values to both Foo handles
-      await fooStore.set({id: 1, rawData: {name: 'the-FOO'}});
+      await fooStore.set(new fooStore.entityClass({name: 'the-FOO'}));
       const fooStore2 = await arc.createStore(fooStore.type, undefined, 'test:3') as SingletonStorageProvider;
```
Comment (galganif) [inline src/runtime/tests/description-test.ts]: migrate?

File: src/runtime/tests/arc-test.ts (hunk 1 above)
Comment (csilvestrini) [reply src/runtime/tests/arc-test.ts]: These didn't work for some reason. It looked complicated :-/

File: src/runtime/tests/description-test.ts (hunk 2 above)
Comment (csilvestrini) [reply src/runtime/tests/description-test.ts]: Same here

**Pull request merged**